pom.xml

```diff
@@ -144,14 +143,10 @@
         <dependency>
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-network-store-client</artifactId>
-            <!-- FIXME: to be removed at next powsybl-ws-dependencies upgrade  -->
-            <version>${powsybl-network-store.version}</version>
         </dependency>
         <dependency>
```

Comment:
You can remove the whole dependency I think. This is used when we need to override the network-store-version. 
But we don't need it anymore when we're based directly on the gridsuite-dependencies versions

Reply:
No I can't. It is used in the repo and if I don't put it in the dependency section, it doesn't matter if it is in gridsuite-dependencies it won't be used.

Reply:
But the remarke is true for another dependency network-store-iidm-impl (I removed it)